### PR TITLE
updated Swish package to include custom header logic

### PR DESCRIPTION
**Pull request by haIIux**
## Description

This PR edits the Swish project file to include a custom header logic. If a user clones the project but does not run the `./rename.sh` script, any new file generated will generate with our custom header to include the MIT licensing and generation language for OpenBytes. Alternatively, if a user does run the `./rename.sh` file the new header with their logic.

## Testing Steps

1. Pull changes.
2. Run `./rename.sh`.
3. Open new project in Xcode.
4. *Note* The default header should not be changed.
5. Create a new file.
6. New header should be displayed.

## Areas of Code Impacted

Swish Project -> Project.swift
`./rename.sh` rename of xcodegen to swish

## Other Notes/Considerations

N/A

### CODE REVIEW

**Comment by 0xLeif** [comment]
Showing ___ORGANIZATIONNAME___

File: Swish/Sources/ProjectLib/Project.swift [hunk 1]
```diff
@@ -1,8 +1,60 @@
...
+                                                //  ___PROJECTNAME___ template generated by OpenBytes on ___DATE___.
+                                                //  Copyright (c) ___YEAR___ OpenBytes
+                                                //
+                                                //  Permission is hereby granted, free of charge, to any person obtaining a copy
+                                                //  of this software and associated documentation files (the "Software"), to deal
+                                                //  in the Software without restriction, including without limitation the rights
+                                                //  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+                                                //  copies of the Software, and to permit persons to whom the Software is
+                                                //  furnished to do so, subject to the conditions found at the following link:
+                                                //  https://github.com/0xOpenBytes/ios-base/blob/main/LICENSE
```
Comment (0xLeif) [inline Swish/Sources/ProjectLib/Project.swift]: This will be updated

Comment (0xLeif) [reply Swish/Sources/ProjectLib/Project.swift]: Fixed in https://github.com/0xOpenBytes/ios-base/pull/20/commits/f8267216f8e5232095f37c7a373476af89bf07f9

**Comment by 0xLeif** [comment]
> Showing _**ORGANIZATIONNAME**_

Removed this line from the templates.

**Review by DanielSincere — APPROVED** [review]
This PR is huge, I think it looks alright.

**Pull request merged**